client/my-sites/site-settings/start-over/index.jsx

```diff
@@ -1,97 +1,80 @@
-						</ActionPanelFigure>
-						<ActionPanelTitle>{ strings.startOver }</ActionPanelTitle>
-						<p>{
-							this.translate( 'If you want a site but don\'t want any of the posts and pages you have now, our support ' +
-								'team can delete your posts, pages, media, and comments for you.' )
-						}</p>
-						<p>{
-							this.translate( 'This will keep your site and URL active, but give you a fresh start on your content ' +
-								'creation. Just contact us to have your current content cleared out.' )
-						}</p>
-						<p>{
-							this.translate( 'Alternatively, you can delete all content from your site by following the steps here.' )
-						}</p>
-					</ActionPanelBody>
-					<ActionPanelFooter>
-						<Button
-							className="settings-action-panel__support-button is-external"
-							href={ support.EMPTY_SITE }>
-							{ strings.emptySite }
-							<Gridicon icon="external" size={ 48 } />
-						</Button>
-						<Button
-							className="settings-action-panel__support-button"
-							href="/help/contact">
-							{ strings.contactSupport }
-						</Button>
-					</ActionPanelFooter>
-				</ActionPanel>
-			</div>
-		);
-	},
-
-	_updateSite: function() {
-		this.setState( {
-			site: this.props.sites.getSelectedSite()
-		} );
-	},
-
-	_goBack: function() {
-		var site = this.state.site;
-		page( '/settings/general/' + site.slug );
-	}
-
-} );
+export default connect(
+	state => ( {
```

Comment:
I know there is no rules around that, but how do you feel about adding parentheses here for consistency?